x-pack/plugins/transform/public/app/sections/transform_management/components/action_stop/stop_action_name.tsx

```diff
@@ -22,60 +25,78 @@ export const stopActionNameText = i18n.translate(
   }
 );
 
+export const getStopActionDisabledMessage = ({
+  items,
+  capabilities,
+}: {
+  items: TransformListRow[];
+  capabilities: TransformCapabilities;
+}) => {
+  const isBulkAction = items.length > 1;
+
+  const { canStartStopTransform } = capabilities;
+
+  // Disable start for transforms if stats does not exist
```

Comment:
Should be `Disable stop...`

Reply:
Removed here [`82de19d` (#166320)](https://github.com/elastic/kibana/pull/166320/commits/82de19d62bea414640929fd4b106c105a51e1a53)

Reply:
Removed here [`82de19d` (#166320)](https://github.com/elastic/kibana/pull/166320/commits/82de19d62bea414640929fd4b106c105a51e1a53)